### ISSUE TITLE
Accessibility: Crate names should be headings

### ISSUE THREAD

**Issue by jquesada2016** [rust-lang/crates.io#11759]
### Current Behavior

I am blind user who uses VoiceOver on macOS and JAWs, NVDA on Windows. When searching for crates, there is no easy way to jump to the different crates because they are all links, rather than being differentiated semantically somehow.

### Expected Behavior

I expect crates to be headings at an appropriate level to make it easy to jump between crate names without having to go through every unrelated link to find them.

### Steps To Reproduce

1. Open your preferred screen reader
2. Go to crates.io on any accessible browser
3. Search for any crate, such as leptos
4. Try navigation commands to find only crate names
5. Observe that the best you can do is jump by link, where there will be many links between crate namesf

### Environment

- Browser: Any browser compatible with your preferred screen reader.
- OS: Anyf

**Comment by DataTriny** [comment]
Hi @jquesada2016,

I have made the necessary changes in #11999. Search results now all contain a level 3 heading spanning the name and the version of each crate. Once you land on a search result, you can press tab once to get to the button to copy the TOML instructions.

It would be awesome if you were able to test it yourself and give feedback.

Thanks.
